From aef25d85fb0043d8feed8c3287f461c629fc2390 Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <tgamblin@baylibre.com>
Date: Tue, 21 Jul 2026 11:25:31 -0400
Subject: [PATCH] tests: test_font_manager: increase test_fontcache_thread_safe
 timeout

This test seems to pass in a local riscv64 container, but fails when run
on the native runners. It's a single threading test with a 10s timeout,
so double that timeout for our builds. For now this should be considered
inappropriate for upstream until we see the failure on their end too.

Upstream-Status: Inappropriate [native runner specific]

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 lib/matplotlib/tests/test_font_manager.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/matplotlib/tests/test_font_manager.py b/lib/matplotlib/tests/test_font_manager.py
index e9cac7caa..57fd84d38 100644
--- a/lib/matplotlib/tests/test_font_manager.py
+++ b/lib/matplotlib/tests/test_font_manager.py
@@ -344,7 +344,7 @@ def _test_threading():
 def test_fontcache_thread_safe():
     pytest.importorskip('threading')
 
-    subprocess_run_helper(_test_threading, timeout=10)
+    subprocess_run_helper(_test_threading, timeout=20)
 
 
 def test_lockfilefailure(tmp_path):
